diff --git a/lib/esm/puppeteer/cdp/FrameManager.js b/lib/esm/puppeteer/cdp/FrameManager.js
index 2b143667c4068928277827ae1b599fe90ee48412..33a528d143eab33a954775b6bd2aea6231296e77 100644
--- a/lib/esm/puppeteer/cdp/FrameManager.js
+++ b/lib/esm/puppeteer/cdp/FrameManager.js
@@ -175,9 +175,10 @@ export class FrameManager extends EventEmitter {
                     this.#frameTreeHandled?.resolve();
                 }),
                 client.send('Page.setLifecycleEventsEnabled', { enabled: true }),
-                client.send('Runtime.enable').then(() => {
-                    return this.#createIsolatedWorld(client, UTILITY_WORLD_NAME);
-                }),
+                // client.send('Runtime.enable').then(() => {
+                //     return this.#createIsolatedWorld(client, UTILITY_WORLD_NAME);
+                // }),
+                this.#createIsolatedWorld(client, UTILITY_WORLD_NAME),
                 ...(frame
                     ? Array.from(this.#scriptsToEvaluateOnNewDocument.values())
                     : []).map(script => {
